prdoc/pr_6624.prdoc

```diff
@@ -0,0 +1,11 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Use `cmd_lib` instead of `std::process::Command` when using `#[docify::export]`
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      "See issue #6194."
```

Comment:
This description is used in the changelog. Ideally, we'll summarize what #6194 is about in a sentence, or paragraph — no need to go into details.

Eg:
```
Simplified the display of commands and ensured they are tested for chain spec builder's `polkadot-sdk` reference docs.
```